lib/takeaway.rb

```diff
@@ -0,0 +1,65 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_accessor :menu, :dishes_selected, :total_price, :total_price_array, :dishes_available, :dish_price
+
```

Comment:
Many instance variables, most are necessary. For example, total_price should just be a method rather than an instance variable.